src/common/hotbar-store.ts

```diff
@@ -120,8 +120,22 @@ export class HotbarStore extends BaseStore<HotbarStoreModel> {
     return toJS(model);
   }
 
-  getActive() {
-    return this.getById(this.activeHotbarId);
+  getActive(): Hotbar {
+    const hotbar = this.getById(this.activeHotbarId);
+
+    if (hotbar) {
+      return hotbar;
+    }
+
+    runInAction(() => {
```

Comment:
1. When you are using `getActive` you don't expect it to cause change. Function should never do anything else than the name of function says. `get` shouldn't `set` anything. 


2. Why is this required for turning strict mode on? I'd expect all of the changes in this PR to be typing or code style related and this is change in functionality.

Reply:
1. Because there MUST always be a hotbar
2. I did this because of point one is now enforced by the compiler and I did this to ensure that it is true. I will review this and see if I can remove this `set`.

Reply:
Sure there might be requirement that there must always be a hotbar, but that doesn't change the fact that function named `get*` shouldn't do anything else than getting something. Developer has to be able to trust that name of function tells everything that it does and it doesn't cause any expected side-effects.